spec/ddtrace/contrib/concurrent_ruby/integration_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'concurrent/future'
+
+require 'spec_helper'
+require 'ddtrace'
+
+RSpec.describe Datadog::Contrib::ConcurrentRuby::Integration do
+  let(:tracer) { ::Datadog::Tracer.new(writer: FauxWriter.new) }
+  let(:configuration_options) { { tracer: tracer } }
+
+  around do |example|
+    unmodified = ::Concurrent::Future.dup
+    Datadog.registry[:concurrent_ruby].patcher.instance_variable_set(:@done_once, {})
+    example.run
+    ::Concurrent.send(:remove_const, :Future)
+    ::Concurrent.const_set('Future', unmodified)
+  end
+
+  subject(:deferred_execution) do
+    outer_span = tracer.trace('outer_span')
+    inner_span = nil
```

Comment:
Just curious; why does this need to be set to `nil` here?

Reply:
To be able to set the variable in the Closure below and make it accessible outside